fabric-render-data-v2/src/main/resources/fabric-render-data-v2.mixins.json

```diff
@@ -0,0 +1,12 @@
+{
+  "required": true,
+  "package": "net.fabricmc.fabric.mixin.renderdata",
+  "compatibilityLevel": "JAVA_16",
```

Comment:
I'm not sure this is correct, someone would have to check (the compatibility level setting is a mess in any case).